src/names.c

```diff
@@ -820,13 +866,13 @@ struct name_repair_opts new_name_repair_opts(r_obj* name_repair,
     .fn = r_null,
     .name_repair_arg = name_repair_arg,
     .quiet = quiet,
-    .call = call
+    .call = call,
```

Comment:
Purposeful?

Reply:
I don't see the new `.frame` struct element being used anywhere?

Reply:
It is used in `new_repair_error_info()` to create an internal error call.

Reply:
Yes, but I don't see where you ever set the `frame` element when you create a new `struct name_repair_opts`

Reply:
oh yeah what happened to that arg. Hmm I'll just call `peek_frame()`

Reply:
but what if `vec_as_names()` was called from C :-(
This stuff is too hard.